src/transaction/log_replication.cpp

```diff
@@ -233,10 +247,18 @@ namespace cublog
 	    break;
 	  }
 	  case LOG_COMMIT:
+	    if (m_replicate_mvcc)
+	      {
+		m_replicator_mvccid->complete_mvccid (header.trid, true);
```

Comment:
Maybe add `const bool commited_transaction = true` or a comment so it is a bit easier to understand what the second argument stands for.

Reply:
Correct, there should be some constants.